src/DynamoCoreWpf/Views/Core/WorkspaceView.xaml

```diff
@@ -34,8 +34,9 @@
                 <Setter.Value>
                     <ControlTemplate TargetType="{x:Type ContextMenu}">
                         <StackPanel Margin="8,0,8,8"
-                                    Width="250">
-                            <ui:InCanvasSearchControl DataContext="{Binding InCanvasSearchViewModel}" />
+                                    Name="ContextMenuPanel">
+                            <ui:InCanvasSearchControl DataContext="{Binding InCanvasSearchViewModel}"
+                                                      Width="{Binding ElementName=ContextMenuPanel, Path=ActualWidth}" />
```

Comment:
No.
Even if we use Grid, width for `InCanvasSearchControl` is `Auto`. So, it will take minimum of the whole possible width.
I.e.
![image](https://cloud.githubusercontent.com/assets/8158404/8126851/f7e4307e-10fa-11e5-87b0-143c1ddd4e52.png)